### ISSUE TITLE
Fieldmanager_Datepicker field JavaScript does not work with Gutenberg enabled

### ISSUE THREAD

**Issue by corywebb** [alleyinteractive/wordpress-fieldmanager#713]
We are testing an existing site with Gutenberg, and I discovered that the `Fieldmanager_Datepicker` field doesn't work with Gutenberg enabled.

## Expected functionality
Click the field, and a jQuery UI Datepicker calendar pops up to select the date.

## Actual functionality
Click the field, and no calendar pops up. You can still type the date into field, but there is no calendar available to pick the date.

**Comment by mboynes** [comment]
Thanks for the bug report @corywebb!

Looks like on `document.ready`, the field isn't actually visible. Therefore, [the field isn't initialized](https://github.com/alleyinteractive/wordpress-fieldmanager/blob/10080e822affeea0a71ec717746e153f9a4406ec/js/fieldmanager-datepicker.js#L4).

Relates to #682 and possibly #710.

**Comment by corywebb** [comment]
Cool, thanks for the quick response @mboynes.

What would the solution be? Maybe change it to use `$( window ).load()`?

**Comment by corywebb** [comment]
@mboynes FYI, I was able to resolve the issue from my theme by adding the following:

```
( function( $ ) {

	$( window ).load( function() {
		if ( typeof fm !== "undefined" ) {
			fm.datepicker.add_datepicker();
		}
	} );

} )( jQuery );
```

It doesn't solve it for everyone else, but it works for my purposes. I think if you just change `$( document ).ready` to `$( window ).load` in your code, it will resolve this issue.

**Comment by kevinfodness** [comment]
Fixed via #757
